packages/discord.js/src/client/actions/UserUpdate.js

```diff
@@ -7,27 +7,21 @@ class UserUpdateAction extends Action {
   handle(data) {
     const client = this.client;
 
-    const newUser = data.id === client.user.id ? client.user : client.users.cache.get(data.id);
-    const oldUser = newUser._update(data);
+    if (data.user.id !== client.users.me.id) return { old: null, updated: null };
```

Comment:
Why couldn't the `userUpdate` event handle normal users ?

Reply:
This is a safety net, the check shouldn't ever be false given discord only emits USER_UPDATE for the client user